client/package.json

```diff
@@ -40,7 +40,8 @@
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
     "test": "react-app-rewired test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "postinstall": "npm rebuild node-sass"
```

Comment:
yeah ok i hope to remove this asap but need to refactor frontend componetns a bit